src/API/WesternStatesWater.WestDaat.Database/EntityFramework/WaterConservationApplication.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace WesternStatesWater.WestDaat.Database.EntityFramework;
+
+public class WaterConservationApplication
+{
+    public WaterConservationApplication()
+    {
+
+    }
+
+    public Guid Id { get; set; }
+
+    public Guid ApplicantUserId { get; set; }
+
+    public DateTimeOffset SubmittedDate { get; set; }
+
+    public int CompensationRateDollars { get; set; }
+
+    public Common.DataContracts.CompensationRateUnits CompensationRateUnits { get; set; }
+
+    public string ApplicationDisplayId { get; set; }
+
+    public string WaterRightNativeId { get; set; }
+
+    public Guid OrganizationId { get; set; }
```

Comment:
```suggestion
    public Guid FundingOrganizationId { get; set; }
```

I think I like this name better for the column / property. It makes it a little more clear

Reply:
updating